server/system/aws.js

```diff
@@ -14,12 +14,14 @@ const _sendAwsRequest = request =>
     })
   })
 
-const inviteUser = email => {
+const inviteUser = async (email, temporaryPassword) => {
```

Comment:
you should add "await" before _sendAwsRequest or remove the "async" keyword

Reply:
Removed the asyncs